index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello World</title>
+    <link rel="stylesheet" href="/style.css">
```

Comment:
- [OPTIONAL] For your css tyles to take effect on the text, you need to remove the `/` from how you reference your style file. Kindly change it to `<link rel="stylesheet" href="style.css">`